ansys/dpf/core/misc.py

```diff
@@ -77,6 +118,17 @@ def find_ansys():
     >>> path = find_ansys()
 
     """
+    versions = [key[-3:] for key in os.environ.keys() if "AWP_ROOT" in key]
+    for version in sorted(versions, reverse=True):
+        if not version.isnumeric():
+            continue
+        if version == __ansys_version__:
```

Comment:
@PProfizi in my opinion this check is not necessary (for next PRs)

Reply:
@cbellot000 removed